rest_api/pipeline/pipelines.yaml

```diff
@@ -1,19 +1,36 @@
 version: '0.7'
 
-components:    # define all the building-blocks for Pipeline
-  - name: ElasticsearchDocumentStore
-    type: ElasticsearchDocumentStore
-    params:
-      host: localhost
+components:    # define all the building-blocks for Pipelines
   - name: ESRetriever
     type: ElasticsearchRetriever
     params:
-      document_store: ElasticsearchDocumentStore    # params can reference other components defined in the YAML
+      document_store: QueryElasticsearchDocumentStore 
       top_k: 5
+  - name: QueryElasticsearchDocumentStore    # for storing normal docs
+    type: ElasticsearchDocumentStore
+    params:
+      host: elasticsearch
+  - name: ElasticsearchDocumentStore         # for storing faq docs
```

Comment:
Currently, there's no way to upload FAQs, so many users may not have a separate index for FAQs.

We could consider using the same document store for both pipelines?

Reply:
Actually, there is a way to upload FAQs, per the explanation in https://github.com/deepset-ai/haystack/blob/master/tutorials/Tutorial4_FAQ_style_QA.ipynb